### PR TITLE
Multiple connect and KSQL clusters

### PR DESCRIPTION
**Pull request by domenicbove**
# Description

This change will allow the ability to confiigure multiple connect clusters out of the same hosts.yml file. It is important that things function as they used to if you only want one cluster. Documentation will need to be updated AND c3 needs to be validated w the multi connect config now

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: hosts_example.yml [hunk 1]
```diff
@@ -178,12 +178,44 @@ schema_registry:
...
       # schema_registry_kerberos_principal: The principal configured in kdc server ex: schemaregistry/ip-172-31-34-246.us-east-2.compute.internal@REALM.EXAMPLE.COM>
+
+
 kafka_connect:
-  hosts:
-    ip-172-31-34-246.us-east-2.compute.internal:
+  children:
+    syslog1:
+      vars:
+        kafka_connect_group_id: connect-syslog1
```
Comment (domenicbove) [inline hosts_example.yml]: whats the best way of documenting this...?

File: roles/confluent.control_center/templates/control-center.properties.j2 [hunk 2]
```diff
@@ -64,16 +64,23 @@ confluent.controlcenter.schema.registry.ssl.key.password={{control_center_keysto
...
 {% set connect_servers = groups.get('kafka_connect', []) %}
 {% if connect_servers %}
 # Kafka Connect Configuration
+{% if kafka_connect_cluster_ansible_group_names is defined %}
+{% for ansible_group in kafka_connect_cluster_ansible_group_names %}
+{% for host in groups[ansible_group] %}{% if loop.index == 1%}confluent.controlcenter.connect.{{hostvars[host]['kafka_connect_group_id']}}.cluster={% endif %}{% if loop.index > 1%},{% endif %}{{ kafka_connect_http_protocol }}://{{ host }}:{{ kafka_connect_rest_port }}{% endfor %}
+
+{% endfor %}
+{% else %}
 confluent.controlcenter.connect.default.cluster={% for host in groups['kafka_connect'] %}{% if loop.index > 1%},{% endif %}{{ kafka_connect_http_protocol }}://{{ host }}:{{ kafka_connect_rest_port }}{% endfor %}
```
Comment (domenicbove) [inline roles/confluent.control_center/templates/control-center.properties.j2]: seems the connect cluster name is hardcoded to default rn

File: roles/confluent.kafka_connect/defaults/main.yml [hunk 3]
```diff
@@ -20,6 +20,13 @@ kafka_connect_service_environment_overrides:
 
 kafka_connect_default_internal_replication_factor: 3
 
+
+kafka_connect_group_id: connect-cluster
+#all necessary topics, should concat the group id
+# config.storage.topic=connect-syslog3-config
```
Comment (domenicbove) [inline roles/confluent.kafka_connect/defaults/main.yml]: remove commented out code

File: hosts_example.yml (hunk 1 above)
Comment (prestonmcgowan) [reply hosts_example.yml]: I added some similar documentation for rack.id into hosts.rack in the following PR.
https://github.com/domenicbove/cp-ansible/pull/7

File: hosts_example.yml [hunk 4]
```diff
@@ -193,27 +194,83 @@ schema_registry:
...
       # kafka_rest_kerberos_principal: The principal configured in kdc server ex: restproxy/ip-172-31-34-246.us-east-2.compute.internal@REALM.EXAMPLE.COM>
+
 ksql:
   hosts:
     ip-172-31-37-15.us-east-2.compute.internal:
       ## For kerberos sasl protocol, EACH host will need these two variables:
       # ksql_kerberos_keytab_path: <The path on ansible host to keytab file, eg. /tmp/keytabs/ksql-ip-172-31-37-15.us-east-2.compute.internal
       # ksql_kerberos_principal: The principal configured in kdc server ex: ksql/ip-172-31-37-15.us-east-2.compute.internal@REALM.EXAMPLE.COM>
+
+# ## To configure multiple ksql clusters, make use of child groups and follow example below
```
Comment (domenicbove) [inline hosts_example.yml]: follow the example below

File: hosts_example.yml [hunk 5]
```diff
@@ -193,27 +194,83 @@ schema_registry:
...
+# # Decide on a name each ksql cluster and use that as ansible group name, this will be how the cluster shows up in c3
```
Comment (domenicbove) [inline hosts_example.yml]: this is how the cluster will be named in c3

File: hosts_example.yml [hunk 6]
```diff
@@ -193,27 +194,83 @@ schema_registry:
...
+# ksql1:
+#   vars:
+#     # The below is a mandatory variable that must be unique for each ksql cluster
```
Comment (domenicbove) [inline hosts_example.yml]: it should end in an underscore by convention

File: hosts_example.yml [hunk 7]
```diff
@@ -193,27 +194,83 @@ schema_registry:
...
+#     ksql2:
+
+kafka_connect:
+  hosts:
+    ip-172-31-34-246.us-east-2.compute.internal:
+      ## For kerberos sasl protocol, EACH host will need these two variables:
+      # kafka_connect_kerberos_keytab_path: <The path on ansible host to keytab file, eg. /tmp/keytabs/connect-ip-172-31-34-246.us-east-2.compute.internal
+      # kafka_connect_kerberos_principal: The principal configured in kdc server ex: connect/ip-172-31-34-246.us-east-2.compute.internal@REALM.EXAMPLE.COM>
+
+# ## To configure multiple connect clusters, make use of child groups and follow example below
```
Comment (domenicbove) [inline hosts_example.yml]: follow the example below

**Pull request merged**